components/brave_wallet_ui/common/hardware/ledgerjs/ledger-untrusted-transport.ts

```diff
@@ -31,16 +32,25 @@ export class LedgerUntrustedMessagingTransport extends LedgerMessagingTransport
     this.addCommandHandler<SignTransactionResponsePayload>(LedgerCommand.SignTransaction, this.handleSignTransaction)
   }
 
-  handleUnlock = async (commandx: UnlockCommand): Promise<UnlockResponsePayload> => {
-    if ((await this.authorizationNeeded())) { // TODO should try / catch this, possible a device error occurs
-      const error = {
+  handleUnlock = async (command: UnlockCommand): Promise<UnlockResponsePayload> => {
+    let payload: LedgerResponse | HardwareOperationResult
```

Comment:
payload can be defined as constant
const isAuthNeeded = await this.authorizationNeeded()
const payload: LedgerResponse | HardwareOperationResult = isAuthNeeded ? { ... } : { ... }